core/src/server.js

```diff
@@ -11,6 +11,11 @@ const server = http.createServer(app);
 const port = config.port;
 server.listen(port, () => {
   console.log(`Soul is running on port ${port}...`);
+
+  if (config.startWithStudio) {
+    // Importing this would do the job.
```

Comment:
Can you explain why importing it does the job?

Reply:
if you build soul-studio with `vite build` and check the `build/` folder you may found that `./build/index.js` starts and exports an HTTP server.